include/swupd.h

```diff
@@ -258,6 +258,10 @@ extern int get_dirfd_path(const char *fullname);
 extern int verify_fix_path(char *targetpath, struct manifest *manifest);
 extern void set_local_download(void);
 extern struct list *files_from_bundles(struct list *bundles);
+extern int system_argv(char *const argv[]);
+extern int system_argv_fd(char *const argv[], int newstdin, int newstdout, int newstderr);
+extern int system_argv_pipe(char *const argvp1[], int stdinp1, int stderrp1,
```

Comment:
@jrguzman-intel I'm okay with using the signatures from your patch, but can you add docstrings where they are defined in helpers.c? I think having good documentation of the expected use (and limitations) of these helper functions will suffice for now.